vars/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 # vars file for openvpn
 openvpn_config_file: "openvpn_{{ openvpn_proto }}_{{ openvpn_port }}"
+openvpn_client_template: "client.ovpn.j2"
```

Comment:
One thing: Put this in `defaults/main.yml` - this is a role default as per https://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable.